src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(/src/images/star.svg);
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--5 .stars__star {
```

Comment:
The selector `.stars--5 .stars__star` applies the active star image to all stars in the group, which is consistent with the intended design. However, for consistency with other selectors, consider using `.stars--5 :nth-child(-n + 5)` to explicitly indicate the number of active stars, even though it results in the same styling.